src/test/java/uk/gov/hmcts/divorce/notification/NotificationDispatcherTest.java

```diff
@@ -166,6 +166,32 @@ void shouldSendNOCForNewDigitalSolicitorNewOrg_Applicant2() {
         verify(applicantNotification).sendToApplicant2Solicitor(caseData, caseId);
     }
 
+    @Test
+    void shouldSendNOCForNewDigitalSolicitorNewOrg_WhenApplicant1EmailIsPresent() {
+        final long caseId = 12345L;
+        final CaseData caseData = CaseData.builder().applicant1(Applicant.builder().email("test@test.com").build()).build();
+        final CaseData previousCaseData = new CaseData();
+        final boolean isApplicant1 = true;
+        final NoticeType noticeType = NoticeType.NEW_DIGITAL_SOLICITOR_NEW_ORG;
+
+        notificationDispatcher.sendNOC(applicantNotification, caseData, previousCaseData, caseId, isApplicant1, noticeType);
+
+        verify(applicantNotification).sendToApplicant1Solicitor(caseData, caseId);
+    }
+
+    @Test
+    void shouldSendNOCForNewDigitalSolicitorNewOrg_WhenApplicant2EmailIsPresent() {
+        final long caseId = 12345L;
+        final CaseData caseData = CaseData.builder().applicant2(Applicant.builder().email("test@test.com").build()).build();
+        final CaseData previousCaseData = new CaseData();
+        final boolean isApplicant1 = false;
+        final NoticeType noticeType = NoticeType.NEW_DIGITAL_SOLICITOR_NEW_ORG;
+
+        notificationDispatcher.sendNOC(applicantNotification, caseData, previousCaseData, caseId, isApplicant1, noticeType);
+
+        verify(applicantNotification).sendToApplicant2Solicitor(caseData, caseId);
+    }
```

Comment:
Do we need to update these tests? They look the same as the tests from line 144 to 167, and don't test that sendToApplicant1/sendToApplicant1Offline/sendToApplicant2/sendToApplicant2Offline are called correctly

Reply:
Sorry i forgot to add those. These tests i added mainly to increase coverage for applicants where email is present but forgot to verify the calls to individual methods